crates/tracing/src/lib.rs

```diff
@@ -0,0 +1,220 @@
+//! Miralis ABI
+//!
+//! While miralis forwards standard SBI calls to the virtualized firmware, miralis do expose its own
+//! ABI for the firmware to interact with.
+//!
+//! The Miralis ABI tries to be compatible with the SBI specification as best as it can.
+//! See: https://github.com/riscv-non-isa/riscv-sbi-doc
```

Comment:
This is a copy paste from another library, it doesn't explain what this library is doing nor how it should be used. The current doc comment might cause confusion and make the crate less maintainable.